ui/selectors/multichain.ts

```diff
@@ -463,9 +477,21 @@ export function getMultichainConversionRate(
   state: MultichainState,
   account?: InternalAccount,
 ) {
-  const { ticker } = getMultichainProviderConfig(state, account);
+  const { conversionRates } = state.metamask;
+  const { chainId } = getMultichainNetwork(state, account);
+
+  const conversionRate = getConversionRatesForNativeAsset({
+    conversionRates,
+    chainId,
+  })?.rate;
 
   return getMultichainIsEvm(state, account)
     ? getConversionRate(state)
-    : getMultichainCoinRates(state)?.[ticker.toLowerCase()]?.conversionRate;
+    : conversionRate;
 }
+
+export const getMultichainConversionRateSelector = createSelector(
+  (state) => state,
+  (_state, account) => account,
+  (state, account) => getMultichainConversionRate(state, account),
+);
```

Comment:
not needed ? Especially that it only gets the conversion rate for the native asset ?

Reply:
Agree that this is also to be removed;
cc @salimtb